insta-ui/src/lib/types.ts

```diff
@@ -1,26 +1,34 @@
+// src/lib/types.ts
+
 /**
  * LLM related types
  */
 
 interface TextContent {
-  type: string;
+  type: 'text';
   text: string;
 }
 
 interface ImageContent {
-  type: string;
+  type: 'image';
```

Comment:
should be `image_url`? 🤔
<img width="728" alt="image" src="https://github.com/user-attachments/assets/92b13ba4-d767-4d2f-8d2f-42a4365fc974">

Reply:
there's some error on the front end after I changed the type to image_url. 
can refer to the lint report

Reply:
Got it, thx! It looks like sth related to the deployment env. I'll see if I can fix this today and merge this PR to the branch